ansible/roles/test/tasks/mtu.yml

```diff
@@ -9,6 +9,12 @@
 - fail: msg="testbed_type {{testbed_type}} is invalid."
   when: testbed_type not in ['t1-lag', 't1', 't1-64-lag']
 
+- set_fact: mtu=0
```

Comment:
You can make this 9100 and remove the default from python script.  Else it conveys the meaning that the test is run with mtu=0. What do you think?

Reply:
I think so. I also a little bit want to keep the main flow within py for the coming pytest.
Whether you mean just make it clear on this issue?
If so. I will do it.

Reply:
Agree. To make it clear.
Done and test passed.